paasta_tools/cli/cli.py

```diff
@@ -84,7 +99,49 @@ def add_subparser(command, subparsers):
     add_subparser_fn(subparsers)
 
 
-def get_argparser():
+PAASTA_SUBCOMMANDS = {
```

Comment:
In the long term we should just standardise the naming for this rather than having to maintain a mapping. I think start-stop-restart needs to be pulled apart, but otherwise it's just s/_/- right?

Reply:
yep, i wanted to reduce the diff and impact, so i didn't pull start/stop/restart apart and didn't rename pause_service_autoscaler command.